services/shared/network-configuration/test/test.ts

```diff
@@ -0,0 +1,80 @@
+import { assert } from 'chai'
+import { Machine, Network } from '../network-configuration'
+import { describe, it } from 'mocha'
+
+describe('Machine', () => {
```

Comment:
Can we separate these into machine-test.js and netwrok-test.js? I have a feeling that their test will be changing quiet frequently and not always together.

Reply:
Yes good idea